bin/s2nd.c

```diff
@@ -365,6 +367,8 @@ int handle_connection(int fd, struct s2n_config *config, struct conn_settings se
         s2n_setup_external_psk_list(conn, settings.psk_optarg_list, settings.psk_list_len),
         "Error setting external psk list");
 
+    GUARD_RETURN(early_data_recv(conn), "Error receiving early data");
```

Comment:
Add a new line: 

```suggestion
    GUARD_RETURN(early_data_recv(conn), "Error receiving early data\n");
```

Reply:
Do not add a newline: https://github.com/aws/s2n-tls/blob/a862c877a9e101d010c08375675897afa50f7a89/bin/echo.c#L38-L42